### DISCUSSION TITLE
Translation of the application name: Yes or No?

### DISCUSSION THREAD

**Discussion by Stheel83** [sqlitebrowser/sqlitebrowser#3530] [General]
Hi

What do you prefer: 
translate the App Name or let it stay in english?
I made a comparison and now i am unsure how you procede:
Example german: DB Browser für SQLite

    localised: de, pl, ar, cs, fr, id, nl, pt_BR, ru, sv, tr (still the old name "SQLite Browser"), uk_UA
    english: it, es, ja, ko_KR, zh, zh_TW

My oppinion: it is a name and it should stay in english. 
What do you think?

**Comment by mgrojo** [comment]
I think it's better to consider it a brand and not translate it.

**Reply by hchiper** [reply]
I think also it should say in English, because it is its name. Since I am working on the French translation, I'll restore the name in English.

But if one choose to translate, why only the word "for" and not "Browser", or even the abbreviation "DB"? That would make its name unrecognizable. I think of "Microsoft Office" for example: nobody calls it "Bureau Microsoft" in French, "Microsoft kantoor" in Dutch, "Microsoft Büro" in German... It would be Funny, wouldn't it?

**Reply by justinclift** [reply]
> Microsoft Büro

For places like Microsoft, that could even have legal consequences.  They'd probably have to register each of the translated names as trademarks, and do so in every jurisdiction around the world.  So every jurisdiction would then have the complete set of trademarks listed, not just the one for the translation in the local language.

Anyway, it's a funny thought. :wink:

**Reply by lucydodo** [reply]
That's fair point. I think I translated the app name in the Korean translation, but I'll change that in next release(`v3.13.1`).
